src/lib/warn-upload-overwrite.js

```diff
@@ -22,6 +24,25 @@ const getEnvironmentKey = () => {
   return `${parsed.hostname}${path}`;
 };
 
+const getCompressibleTypes = async () => {
+  const parsedUrl = url.parse(environment.apiUrl);
+  const configUrl = `${parsedUrl.protocol}//${parsedUrl.auth}@${parsedUrl.host}/api/couch-config-attachments`;
+  try {
+    if (cache.has('compressibleTypes')) {
+      return cache.get('compressibleTypes');
+    }
+    const resp = await request.get({ url: configUrl, json: true });
+    cache.set('compressibleTypes', resp.compressible_types);
+    return resp.compressible_types;
+  } catch(e) {
+    cache.set('compressibleTypes', null);
+    e.statusCode === 404 ?
```

Comment:
I think we should only set cache to null when the response is a 404, and allow a retry for other failures.